src/io/flutter/FlutterBundle.properties

```diff
@@ -48,3 +48,8 @@ update.module.type=Update Module Type
 updating.module.type.requires.project.reload.proceed=Updating module type requires project reload. Proceed?
 
 waiting.for.flutter=Waiting for debug connection...
+
+path.to.dart.file.not.set=path to Dart source not specified
+dart.file.not.found=Dart file not found: {0}
+not.a.dart.file.or.directory=Specified file is not a Dart file or directory: {0}
+work.dir.does.not.exist=Working directory doesn''t exist: {0}
```

Comment:
I think this is to escape the quote? This is what it looks like in the dart properties file.